ods_ci/tests/Resources/CLI/ModelRegistry/samples/istio/components/istio.env

```diff
@@ -1,4 +1,5 @@
 AUTH_PROVIDER=opendatahub-auth-provider
+DOMAIN=
```

Comment:
Does it works without any value for key Domain?

Reply:
Yes, the operator sets that now.